datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -58,6 +58,6 @@ selectors34==1.2.0; sys_platform == 'win32' and python_version < '3.4'
 serpent==1.27; sys_platform == 'win32'
 simplejson==3.6.5
 six==1.12.0
-supervisor==3.3.3; python_version < '3.0'
+supervisor==4.0.0
```

Comment:
[`supervisor==4.0.1`](https://pypi.org/project/supervisor/4.0.1/)

Reply:
Done, thanks for the heads-up!